conf/messages

```diff
@@ -53,7 +53,7 @@ vatDetails.checkWhatYouOwe = Check what you owe and make a payment
 returnObligation.noReturn = No returns due right now
 returnObligation.nextReturn = Next return due
 returnObligation.viewReturns = View return deadlines
-returnObligation.submit = Submit return
+returnObligation.submit = Submit VAT return
```

Comment:
Got a failing test because of this - please could you change back (as its being done in BTAT-5917 :) )